CONTRIBUTION.md

```diff
@@ -64,3 +64,16 @@ Bug: T42
 The first line is the subject and should be no longer than 50 characters, the second line is always blank, and other
 lines should be wrapped at 80 characters. This allows the message to be easier to read on GitHub as well as in various
 git tools.
+
+### PR workflow
+
+- Before submitting a PR
+  - Format your code and fix all the spelling/grammatical mistakes.
+  - Limit the column length to 120 characters.
+  - Error messages should not be more than 2 lines long.
```

Comment:
I think what @aravindhp meant is single line here. The error messages should not be more than 1 line but I'll let him speak for it

Reply:
Yes, I meant single line here.